notebooks/data-sources/TestGrid/metrics/probability_to_fail.ipynb

```diff
@@ -0,0 +1,1191 @@
+{
```

Comment:
<p>Just curious, why don't you normalize the probabilities here?</p><p> </p><p>Also doesn't <code>probability = True</code> normalize them?</p><pre class="ql-syntax" spellcheck="false">Line #14.    model = SVC(probability=True).fit(trainx, trainy)
</pre><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aicoe-aiops/ocp-ci-analysis/pull/310/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='17'/>

Reply:
No, it actually enables the probability estimates(from the methods `predict_proba` and `predict_log_proba`) because SVMs do not directly provide probability estimates.

Reply:
I'm a little confused. What is the difference between a "normalized probability" and "probability estimates" ?

Reply:
Probability estimates here are the calibrated probabilities from SVM and they are different from normalized probabilities in the sense that their summation is not equal to 1. The output of the predict_proba method can be directly interpreted as a confidence level, i.e. among the samples to which it gave a predict_proba value close to 0.8, approximately 80% actually belong to the positive class.